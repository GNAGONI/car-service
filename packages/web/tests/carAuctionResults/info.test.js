import React from 'react'
import renderer from 'react-test-renderer'
import Info from '../../components/carAuctionResults/commonCollapse'

it('renders correctly', () => {
  const tree = renderer.create(<Info />).toJSON()
  expect(tree).toMatchSnapshot()
})
