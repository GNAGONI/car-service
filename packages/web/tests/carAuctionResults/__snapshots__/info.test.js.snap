// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="content section-info common-collapse"
>
  <div
    aria-expanded={null}
    className="mb-4 collapse"
  >
    
  </div>
  <div
    className="btn-block text-center pt-2"
  >
    <a
      className="default read-more "
      onClick={[Function]}
    >
      <span
        className="more"
      >
        Read more
      </span>
    </a>
  </div>
</section>
`;
